### ISSUE TITLE
Licensing issues

### ISSUE THREAD

**Issue by char0n** [facebook/prop-types#162]
The latest version of prop-types 15.6.0 on [npm](https://www.npmjs.com/package/prop-types) contains files `prop-types.js` and `prop-types.min.js`. These files represents UMD builds. Unfortunately `prop-types.js` file contains segments still licensed under BSD + Patents. It can be seen for example here:

```js
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 *
 * 
 */

function makeEmptyFunction(arg) {
  return function () {
    return arg;
  };
}
```

I think this happened accidentally by creating the UMD build before changing the license headers in the required files. There are totally 3 references to the BSD + Patents and it can be seen [here](https://source.tonicdev.net/npm/prop-types/prop-types.js?engine=8.x.x&t=1519229510692). This may have some legal implications for people using it.

Is it possible to release a patch release fixing this UMD build files ?

Thank you very much

**Comment by aweary** [comment]
@char0n thanks for the heads up, I verified that building from master does not include the outdated BSD header. cc @sophiebits

**Comment by char0n** [comment]
@aweary is it possible to publish a new release with the fixed bundle ? We just cannot use prop-types until it contains the BSD + Patent references.

**Comment by aweary** [comment]
@char0n I think we could push a patch release. I'll ping the right people to look into this today, sorry about the delay.

**Comment by sophiebits** [comment]
@char0n You can always build it locally if you want.

**Comment by aweary** [comment]
@char0n I've published 15.6.1 and verified that [`prop-types.js`](https://unpkg.com/prop-types@15.6.1/prop-types.js) no longer contains any BSD license headers. Thanks again for bringing this to our attention!

**Comment by char0n** [comment]
Yeah I double checked. Thank for for swift response and release.
